### ISSUE TITLE
[4.1.0 ?][Libretro core] Wolfenstein 3D

### ISSUE THREAD

**Issue by mgoulartBR** [recalbox/recalbox-os#835]
Hi guys,

There is a new core for playing Wolfenstein 3D on Retroarch
Is it possible to include it on a future Recalbox release?

https://github.com/libretro/libretro-wolfenstein3d

**Comment by ziviani** [comment]
Good idea mgoulartBR. It will be very good to have Wolfenstein 3D in Recalbox

**Comment by digitalLumberjack** [comment]
We must create a specific system for those one game cores and other ports.

**Comment by mgoulartBR** [comment]
I totally agree digitaLumberjack!

**Comment by ShunLe** [comment]
And créate à emulator with configen for id software game in one systems. Run Wolf4sdl prboom (or chocolaté doom) , tyrquake, ioquake3 and quake2bin ? 3 libretro core and 2 engine. After eduke32 ? Add or not ?
